ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -216,7 +216,7 @@ def run(self):
                 msg = 'BGP route GR timeout: neighbor %s (ASN %s' % (nei, asn)
                 self.fails.add(msg)
 
-        if cli_data['po'][1] > 0:
```

Comment:
What's the expected behavior during the fast reboot? flap only once? If it flaps more than once, should we consider that a failure? thanks.